AndroidSdk/AvdManager/AvdManager.cs

```diff
@@ -279,7 +299,7 @@ void ParseIniFile(Dictionary<string, string> properties, string iniFile)
 			}
 		}
 
-		IEnumerable<string> run(params string[] args)
+		IEnumerable<string> run(string[] args, Action<StreamWriter> inputAction = null)
```

Comment:
I notice this code is basically duplicated 3 times in each tool, so I want to try and refactor that into a separate runner thing that we can just reuse - especially with the "keep posting input until we are done" 10x engineering.

Reply:
I see this is already done via the new ProcessRunner used in adbd. So will just update the old tools in a new PR.